bin/feature

```diff
@@ -210,7 +213,7 @@ when 'merge'
       # rebase the unpushed master commits if any.
       "git rebase --preserve-merges origin/#{dev_branch}",
       # merge the feature branch into master
-      "git merge --no-ff --edit -m #{pull_info[:description].shellescape} \"#{feature}\"",
+      "git merge --no-ff --edit -m #{pull_desc} \"#{feature}\" || #{abort_merge}",
```

Comment:
I like having the `.shellescape` here so it's clear that the variable is properly escaped.